benchmarks/queries/clickbench/README.md

```diff
@@ -28,6 +28,23 @@ SELECT
 FROM hits;
 ```
 
+### Q1
+Query to test distinct count for String. Three of them are all small string (length either 1 or 2).
+
+```sql
+SELECT 
+    COUNT(DISTINCT "HitColor"), COUNT(DISTINCT "BrowserCountry"), COUNT(DISTINCT "BrowserLanguage")
+FROM hits;
+```
+
+### Q2
+Query to test distinct count for String. "URL" has length greater than 8
```

Comment:
Does this query test something signfiicantly different than the Q0 and Q1?

What do you think about maybe adding a GROUP BY as well?

```
❯ SELECT  "BrowserCountry",  COUNT(DISTINCT "HitColor"), COUNT(DISTINCT "BrowserCountry"), COUNT(DISTINCT "BrowserLanguage") FROM 'hits.parquet' GROUP BY 1 ORDER BY 2 DESC LIMIT 10;
+----------------+---------------------------------------+---------------------------------------------+----------------------------------------------+
| BrowserCountry | COUNT(DISTINCT hits.parquet.HitColor) | COUNT(DISTINCT hits.parquet.BrowserCountry) | COUNT(DISTINCT hits.parquet.BrowserLanguage) |
+----------------+---------------------------------------+---------------------------------------------+----------------------------------------------+
| GU             | 3                                     | 1                                           | 2                                            |
| _P             | 3                                     | 1                                           | 3                                            |
| �
                 | 3                                     | 1                                           | 83                                           |
| 4v             | 3                                     | 1                                           | 2                                            |
| nH             | 3                                     | 1                                           | 3                                            |
| nS             | 3                                     | 1                                           | 4                                            |
| MI             | 3                                     | 1                                           | 6                                            |
| 9F             | 3                                     | 1                                           | 2                                            |
| Sc             | 3                                     | 1                                           | 3                                            |
| BO             | 3                                     | 1                                           | 3                                            |
+----------------+---------------------------------------+---------------------------------------------+----------------------------------------------+
10 rows in set. Query took 1.325 seconds.
```

Reply:
Q1 is short string only, Q2 has URL which is long string

Reply:
`SearchPhrase` contains long string

Reply:
I think we can remove Q2, it is similar to Q0.